src/commands/apps-create.ts

```diff
@@ -0,0 +1,208 @@
+  }
+
+  // TODO(caot): Uncomment this after apps:sdkconfig command is implemented
+  // logger.info("");
+  // logger.info("Run this command to print out your config file:");
+  // logger.info(`  firebase apps:sdkconfig ${appMetadata.appId}`);
+}
+
+async function initiateIosAppCreation(options: {
+  project: string;
+  nonInteractive: boolean;
+  displayName?: string;
+  bundleId?: string;
+  appStoreId?: string;
+}): Promise<any> {
+  if (!options.nonInteractive) {
+    await prompt(options, [
+      DISPLAY_NAME_QUESTION,
+      {
+        type: "input",
+        default: "",
+        name: "bundleId",
+        message: "Please specify your iOS app bundle ID:",
+      },
+      {
+        type: "input",
+        default: "",
+        name: "appStoreId",
+        message: "Please specify your iOS app App Store ID:",
+      },
+    ]);
+  }
+  if (!options.bundleId) {
+    throw new FirebaseError("Bundle ID for iOS app cannot be empty");
+  }
+
+  const spinner = ora("Creating your iOS app").start();
+  try {
+    const appData = await createIosApp(options.project, {
+      displayName: options.displayName,
+      bundleId: options.bundleId,
+      appStoreId: options.appStoreId,
+    });
+    spinner.succeed();
+    return appData;
+  } catch (err) {
+    spinner.fail();
+    throw err;
+  }
+}
+
+async function initiateAndroidAppCreation(options: {
+  project: string;
+  nonInteractive: boolean;
+  displayName: string;
+  packageName: string;
+}): Promise<any> {
+  if (!options.nonInteractive) {
+    await prompt(options, [
+      DISPLAY_NAME_QUESTION,
+      {
+        type: "input",
+        default: "",
+        name: "packageName",
+        message: "Please specify your Android app package name:",
+      },
+    ]);
+  }
+  if (!options.packageName) {
+    throw new FirebaseError("Package name for Android app cannot be empty");
+  }
+
+  const spinner = ora("Creating your Android app").start();
+  try {
+    const appData = await createAndroidApp(options.project, {
+      displayName: options.displayName,
+      packageName: options.packageName,
+    });
+    spinner.succeed();
+    return appData;
+  } catch (err) {
+    spinner.fail();
+    throw err;
+  }
+}
+
+async function initiateWebAppCreation(options: {
+  project: string;
+  nonInteractive: boolean;
+  displayName: string;
+}): Promise<any> {
+  if (!options.nonInteractive) {
+    await prompt(options, [DISPLAY_NAME_QUESTION]);
+  }
+  if (!options.displayName) {
+    throw new FirebaseError("Display name for Web app cannot be empty");
+  }
+  const spinner = ora("Creating your Web app").start();
+  try {
+    const appData = await createWebApp(options.project, { displayName: options.displayName });
+    spinner.succeed();
+    return appData;
+  } catch (err) {
+    spinner.fail();
+    throw err;
+  }
+}
+
+module.exports = new Command("apps:create [platform] [displayName]")
+  .description(
+    "create a new Firebase app\n\n" +
```

Comment:
Using newlines in the command description is a bit non-standard. Discussed a change offline.

Reply:
Done